wikilegis/static/javascript/content/events.js

```diff
@@ -68,6 +67,18 @@ function submitEvent(event) {
   if ('commentsForm' in dataset) {
     forms.sendComment(event.target);
   }
+
+  if ('amendmentsForm' in dataset) {
+    forms.sendAmendment(event.target);
+  }
+}
+
+function focusEvent(event) {
+  const dataset = event.target.dataset;
+
+  if ('amendmentsInput' in dataset) {
```

Comment:
You're adding the original text inside the form even when it is not an "Edit" type amendment. "Add" and "Exclusion" types shouldn't have its form auto-filled when focused.